docs/getting-started/mainnet.md

```diff
@@ -0,0 +1,469 @@
+  $ sed -i.bak -E 's#^(create_empty_blocks_interval[[:space:]]+=[[:space:]]+).*$#\1"5s"#' ~/.chain-maind/config/config.toml
+  ```
+
+**Note**: We suggest using `persistent_peers` instead of `seeds` to provide stable state-sync experience.
+##############################[TODO]- seed,genesis - [TODO]##############################
+
+##############################[TODO]- seed,genesis - [TODO]##############################
+### Step 2-3. (**Optional**) Enable STATE-SYNC
+
+This is an optional step, If you would like to build a node with complete blockchain data, jump to [Step 3](#step-3-run-everything).
+
+With  [STATE-SYNC](https://docs.tendermint.com/master/tendermint-core/state-sync.html)  your node will download data related to the head or near the head of the chain and verify the data. This leads to drastically shorter times for joining a network for validator. For **validator**, It will be amazingly fast to sync the near head of the chain and join the network.
+::: warning CAUTION
+Blocks before state-sync `trust height` will **NOT** be queryable. If you want to run a **full node** or a validator with complete blockchain data; It is not suggested to use state-sync. 
+:::
+Follow the below optional steps to enable state-sync:
+- For state-sync configuration, in `~/.chain-maind/config/config.toml`, please modify the configurations under [statesync] `enable`, `rpc_servers`, `trust_height` and `trust_hash` by:
+
+  ```bash
+  $ LATEST_HEIGHT=$(curl -s https://mainnet.crypto.org:26657/block | jq -r .result.block.header.height); \
+  BLOCK_HEIGHT=$((LATEST_HEIGHT - 1000)); \
+  TRUST_HASH=$(curl -s "https://mainnet.crypto.org:26657/block?height=$BLOCK_HEIGHT" | jq -r .result.block_id.hash)
+
+  $ sed -i.bak -E "s|^(enable[[:space:]]+=[[:space:]]+).*$|\1true| ; \
+  s|^(rpc_servers[[:space:]]+=[[:space:]]+).*$|\1\"https://mainnet.crypto.org:26657,https://mainnet.crypto.org:26657\"| ; \
+  s|^(trust_height[[:space:]]+=[[:space:]]+).*$|\1$BLOCK_HEIGHT| ; \
+  s|^(trust_hash[[:space:]]+=[[:space:]]+).*$|\1\"$TRUST_HASH\"| ; \
+  s|^(seeds[[:space:]]+=[[:space:]]+).*$|\1\"\"|" ~/.chain-maind/config/config.toml
+  ```
+##############################[TODO]- update seed,genesis - [TODO]##############################
+## Step 3. Run everything
+
+::: warning CAUTION
+This page only shows the minimal setup for validator node.
+
+In production, you may want to run full nodes
+as sentries (see [Tendermint](https://docs.tendermint.com/master/tendermint-core/running-in-production.html)), restrict your validator connections to only connect to your full nodes, test secure storage of validator keys with [tmkms](https://github.com/iqlusioninc/tmkms) etc.
+:::
+
+### Step 3-1. Create a new key and address
+
+Run the followings to create a new key. For example, you can create a key with the name `Default` by:
+
```

Comment:
I think here we should not suggest the use of chain-maind directly without any secure instructions. Maybe the better way is to redirect them to the mainnet address generation guide: https://crypto.org/docs/wallets/mainnet-address-generation.html

Reply:
Nice suggestion :D

Reply:
Added, see this [link](https://github.com/crypto-org-chain/chain-docs/pull/139/commits/58b41340e73ddf740bc29edb2d74275013516641#diff-a556d8b640301c4c0d15e1f6cfb9b8b1d021a9d56d933eac621cab44a873802cR161)